packages/studiocms_core/src/schemas/config/mdx.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'astro/zod';
+import type { PluggableList } from 'unified';
+
+// Get the type of the remark-rehype options
+type ToHastOptions = import('mdast-util-to-hast').Options;
```

Comment:
Hmmm just wondering why this can't be a regular import?

Reply:
it was arguing about non portable types :sweat_smile: for some reason this prevents that error